_topics/en/ptlc.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Point Time Locked Contracts (PTLCs)
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: ptlc
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Lightning Network
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Point Time Locked Contracts (PTLCs)** are conditional payments that
+  can replace the use of HTLCs in LN payment channels, same-chain
+  coinswaps, some cross-chain atomic swaps, and other contract
+  protocols.  Compared to HTLCs, they can be more private and use less
+  block space.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  PTLCs differ from [HTLCs][topic htlc] in their primary locking and
+  unlocking method:
+
+  - **HTLC hash locks:** are locked using a hash digest and unlocked by
+    providing the corresponding preimage.  The most commonly used hash function is
+    SHA256, which produces a 256-bit (32-byte) digest commonly generated
+    from a 32-byte preimage.
+
+      When used to secure multiple payments (e.g. a routed LN payment or
+      an atomic swap), all payments use the same preimage and hash lock.
+      This creates a link between those payments if they're published
+      onchain or if they're routed offchain though surveillance nodes.
+
+  - **PTLC point locks:** are locked using a public key (a *point* on
+    Bitcoin's elliptic curve) and unlocked by providing a corresponding
+    signature from a satisfied [signature adaptor][topic adaptor
+    signatures].  For a proposed [schnorr signature][topic schnorr
+    signatures] construction, the key would be 32 bytes and the signature
+    64 bytes.  However, using either multiparty ECDSA or schnorr key
+    aggregation and signing, the keys and signature can be combined
+    with other keys and signatures needed to authorize any spend,
+    allowing point locks to use zero bytes of distinct block space.
+
+      Each point lock can use different keys and signatures, so there is
+      nothing about the point lock that correlates different payments
+      either onchain or when routed offchain through surveillance nodes.
+
+  Implementation of PTLCs in Bitcoin requires creating [signature
```

Comment:
We link to adaptor signatures and schnorr sigs back to back just above this, so might be redundant linking.

Reply:
Seems ok to me. I don't have a strong opinion either way.